torch_xla/csrc/tensor.cpp

```diff
@@ -393,6 +423,16 @@ xla::int64 XLATensor::GetNextTensorId() {
   return id_generator->fetch_add(1);
 }
 
+std::pair<ir::NodePtr, bool> XLATensor::GetViewIrNode(View* view) {
```

Comment:
I would use a structure here instead of a pair.

Reply:
Makes sense.